bin/yi_appium_caps_util

```diff
@@ -9,11 +9,14 @@ options = {}
 
 OptionParser.new do |opts|
   opts.banner = "You.i Utility for appium.txt capabilities"
-  puts opts.banner
+  # puts opts.banner
```

Comment:
I took it out because it is printed twice when attempting to show the help
```
ahmadaltamimi@(capsModification):$ ./yi_appium_caps_util -h
You.i Utility for appium.txt capabilities
You.i Utility for appium.txt capabilities
    -u, --update                     update device ID and IP address
    -c, --create PLATFORM            creates the appium.txt file for specefied platform
    -f, --file FILE                  capabilities path and file name (default: ./appium.txt)
    -h, --help                       Prints this help
```
I could put it back if needed